lib/chat/file_index.ex

```diff
@@ -1,53 +1,47 @@
 defmodule Chat.FileIndex do
   @moduledoc "Keeps an index of files and keys it"
 
-  alias Chat.Dialogs.Dialog
-  alias Chat.Utils
-
-  def add_file(key, %Dialog{a_key: a, b_key: b}, message_id, secret) do
-    save(key, a |> Utils.hash(), message_id, secret)
-    save(key, b |> Utils.hash(), message_id, secret)
-  end
-
-  def add_file(key, room_key, message_id, secret) do
-    save(key, room_key |> Utils.hash(), message_id, secret)
-  end
-
-  def list_references(key, %Dialog{a_key: a, b_key: b}) do
-    MapSet.union(
-      list_references(key, Utils.hash(a)),
-      list_references(key, Utils.hash(b))
+  alias Chat.Db
+
+  def get(reader_hash, file_key) do
```

Comment:
Arguments have switched places. Before, `file_key` was the first argument.

Reply:
I've changes that since `:file_index` is reader oriented structure

Reply:
Can you change this line then?
https://github.com/Buckitup-chat/chat/blob/8ce2e0985d1aba65a1a8d5c6f9d337ac9a7f9ea2/lib/chat_web/live/main_live/index.ex#L427